docs/examples.md

```diff
@@ -420,6 +420,65 @@ object PerftConverter extends IOApp.Simple {
 ```
 @:@
 
+## Writing data to a CSV file
+
+If you want to save a list of a case class into a CSV file this helper method may aid you:
+
+@:select(scala-version)
+
+@:choice(scala-3)
+```scala mdoc:reset:silent
+import fs2.io.file.Files
+import fs2.data.csv.*
+import fs2.data.csv.generic.semiauto.*
+import fs2.io.file.Path
+import cats.effect.IO
+import fs2.Pipe
+
+case class YourCaseClass(n: String, i: Int)
+
+given CsvRowEncoder[YourCaseClass, String] = deriveCsvRowEncoder
+
+def writeCaseClassToCsv[A](path: Path)(using CsvRowEncoder[A, String]): Pipe[IO, A, Nothing] =
+    _.through(encodeUsingFirstHeaders(fullRows = true))
+      .through(fs2.text.utf8.encode)
+      .through(Files[IO].writeAll(path))
+      .drain
+
+// Usage
+fs2.Stream.emits(Seq(YourCaseClass("s", 1))).through(writeCaseClassToCsv(Path("temp.csv")))
+
+```
+
+@:choice(scala-2)
+
+```scala mdoc:reset:silent
+import fs2.io.file.Files
+import fs2.data.csv._
+import fs2.data.csv.generic.semiauto._
+import fs2.io.file.Path
+import cats.effect.IO
+import fs2.Pipe
+
+case class YourCaseClass(n: String, i: Int)
+
+implicit val csvRowEncoder: CsvRowEncoder[YourCaseClass, String] = deriveCsvRowEncoder
+
+def writeCaseClassToCsv[A](path: Path)(using CsvRowEncoder[A, String]): Pipe[IO, A, Nothing] =
+    _.through(encodeUsingFirstHeaders(fullRows = true))
+      .through(fs2.text.utf8.encode)
+      .through(Files[IO].writeAll(path))
+      .drain
+
+// Usage
+fs2.Stream.emits(Seq(YourCaseClass("s", 1))).through(writeCaseClassToCsv(Path("temp.csv")))
```

Comment:
I might be asking a silly question, but where is the evaluation happening in these scripts?
They appear to be more similar to code examples than auto-runnable scripts, no?
Or are they meant to be used as scala worksheets (and in that case, where's the Stream evaluation)?
I would add, like in other examples the usual IOApp.Simple and .compile.drain, or briefly explain how to use them as worksheets.